api_app/service_bus/deployment_status_updater.py

```diff
@@ -175,9 +175,9 @@ async def update_status_in_database(self, message: DeploymentStatusUpdateMessage
             # Marking as true as this message will never succeed anyways and should be removed from the queue.
             result = True
             error_string = strings.DEPLOYMENT_STATUS_ID_NOT_FOUND.format(message.id)
-            logging.error(error_string)
-        except Exception as e:
-            logging.error(strings.STATE_STORE_ENDPOINT_NOT_RESPONDING + " " + str(e))
+            logging.exception(error_string)
```

Comment:
I think the error string variable is redundant since it's not used elsewhere. I think there're other places where we have it and can consider using the expression directly.

Reply:
removed